curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9eb4a5d68f12f32d9b440.md

```diff
@@ -9,6 +9,10 @@ dashedName: step-89
 
 Remember that `document.querySelectorAll` returns a `NodeList`, which is array-like but is not an array. However, the `Array` object has a `.from()` method that accepts an array-like and returns an array. This is helpful when you want access to more robust array methods, which you will learn about in a future project.
 
+```js
+const divArray = Array.from(document.querySelectorAll('div'));
```

Comment:
Hi Alagu 👋🏽

This example can be more detailed. Why not something like this:
```suggestion
const str = 'Hello, World!';
const strArray = Array.from(str);

console.log(strArray); // Output: ['H', 'e', 'l', 'l', 'o', ',', ' ', 'W', 'o', 'r', 'l', 'd', '!']


const mySet = new Set([1, 2, 3, 4, 5]);
const setArray = Array.from(mySet);

console.log(setArray); // Output: [1, 2, 3, 4, 5]
```

Reply:
Thank you for your feedback, Chris. I appreciate your review of the pull request.

From my understanding, the code example required here is more inclined to how to use Array.from() function for the NodeList.  Do you think we need to give a brief example of how to use Array.from() using other examples like string/ set?

My suggestion would be to include may be one such example for using Array.from()  for string and other example that uses Array.from() for NodeList.

Looking forward to your insights and further discussion to make changes to the code.

Best regards,
Alagu

Reply:
The code example we want is about how to use `Array.from()`, to show campers that `Array.from()` indeed turns an array-like item into an array. You can see I also provided the output of the two examples.

If you can provide an example that does it for `NodeList `, that would be great too.

Reply:
@Ksound22 

We shouldn't use set because that hasn't been taught yet.
I vote we don't use this first example.

Your second example would work much better.

Reply:
Ok